mothership-ec/cog-mothership-file-manager
# File tag lazy loading

#### What does this do?

Lazy Loading for file tags - breaks BC (getters & setters for tags used, instead of public member)!
#### How should this be manually tested?

Check pages where files are handled, ensure nothing breaks! Check file objects before and after calling `getTags()`, make sure this uses the `FileProxy` class, try updating tags etc.
#### Related PRs / Issues / Resources?

Relies on https://github.com/messagedigital/cog/pull/386
#### Anything else to add? (Screenshots, background context, etc)

Should reduce the number of queries on pages where files are loaded. See file manager.
